### ISSUE TITLE
Reverb applied to input signal when monitoring is turned down

### ISSUE THREAD

**Issue by dyfer** [essej/sonobus#10]
It seems that turning on the reverb on the main output applies it to the input signal even when the input monitoring is turned all the way down. 
However, muting input does mute the signal going to the reverb as well.

**Comment by essej** [comment]
Yes, this was an intentional decision. Because some people who are not using software monitoring (monitoring fully off) might want to still get some reverb. However, I think it’s a flawed compromise. I just need to add a dedicated reverb send control for the input, as well as all the individual connected users.
